### DISCUSSION TITLE
[Feature] Use image upload without importing into Immich

### DISCUSSION THREAD

**Discussion by rishid** [immich-app/immich#6743] [Feature Request]
### The feature

I want to be able to use the mobile upload functionality and external library support of Immich exclusively, skipping the internally managed photo libraries. This is maybe slightly strange request but I have existing workflows and use photo editing tools regularly on my photos. The restriction of being unable to modify/export/move files from the Internal library breaks this flow. 

I would love for the Immich mobile app to do the uploading into ./upload but _not_ import it into a library. I could then use these images/videos, edit them, and then move them into an external library for Immich to manage. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
The app's upload mechanism relies on Immich keeping assets in the library to avoid re-uploading the same asset etc, so this is out of scope.
